js_basics_array/task6.js

```diff
@@ -0,0 +1,5 @@
+const numbersList = [1, 10, 14, 2, 4, 5, 43, 34, -10000, 0, 23, 45.78];
+const cloneArray = Array.from(numbersList);
+cloneArray.sort((firstElement, secondElement) => firstElement - secondElement);
```

Comment:
use shorter names for parameters in callback function. `a` and `b` is kind of standard 
```js
(a, b) => a - b)
```